Cython/Plex/Scanners.py

```diff
@@ -174,26 +170,30 @@ def run_machine_inlined(self):
         buf_len = len(buffer)
         b_action, b_cur_pos, b_cur_line, b_cur_line_start, b_cur_char, b_input_state, b_next_pos = \
             None, 0, 0, 0, u'', 0, 0
+
         trace = self.trace
         while 1:
-            if trace:  #TRACE#
-                print("State %d, %d/%d:%s -->" % (  #TRACE#
-                    state['number'], input_state, cur_pos, repr(cur_char)))  #TRACE#
+            if trace:
+                print("State %d, %d/%d:%s -->" % (
+                    state['number'], input_state, cur_pos, repr(cur_char)))
+
             # Begin inlined self.save_for_backup()
-            #action = state.action #@slow
-            action = state['action']  #@fast
+            # action = state.action   # @slow
+            action = state['action']  # @fast
```

Comment:
I think the slow/fast comments (and the alternative implementations) can all be removed. They were probably added during optimisation, but changing the implementation of "state" would require a very good reason now, with careful benchmarking, regardless of any comments.